agoric/chain.json

```diff
@@ -38,19 +38,24 @@
   },
   "codebase": {
     "git_repo": "https://github.com/Agoric/agoric-sdk/",
-    "recommended_version": "agoric-upgrade-8",
+    "recommended_version": "agoric-upgrade-15",
+    "go_version": "1.20.2",
     "compatible_versions": [
-      "agoric-upgrade-8"
+      "agoric-upgrade-15"
     ],
     "genesis": {
       "genesis_url": "https://main.agoric.net/genesis.json"
     },
     "versions": [
       {
-        "name": "agoric-upgrade-8",
-        "recommended_version": "agoric-upgrade-8",
+        "name": "agoric-upgrade-15",
```

Comment:
Please retain upgrade-8 in the version history

Reply:
Is `versions` expected to be complete? The agoric chain has had [many releases](https://github.com/Agoric/agoric-sdk/releases), and we just realized in the latest one that we've been lax in documenting them here.

Reply:
At the Chain Registry we try to at least retain the history of versions that have ever been registered, but, yes, ideally versions contains a complete history. If it's quite long it can be relocated to the 'versions.json' file.

Reply:
OK, I split out the versions.json file and we will backfill it after this PR lands.